test-app/package.json

```diff
@@ -46,8 +47,8 @@
     "@typescript-eslint/eslint-plugin": "^5.11.0",
     "@typescript-eslint/parser": "^5.11.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-animated": "0.12.0",
-    "ember-animated-tools": "github:ember-animation/ember-animated-tools#1edcfd7f0be0a9b4b69450f0fc4d945f31277215",
+    "ember-animated": "workspace:*",
+    "ember-animated-tools": "file:../ember-animated-tools-0.5.0.tgz",
```

Comment:
As `ember-animated-tools`now  a monorepo, we can't use `github:ember-animation/ember-animated-tools#v2-addon` scheme.

as temp workaround, packed `ember-animated-tools` and used here.

Once `ember-animated-tools` published to npm (via beta or actual release), will have a follow up PR to point back to npm.